tests/unit/pypyr/steps/pype_test.py

```diff
@@ -25,12 +25,14 @@ def test_pype_get_arguments_all():
      use_parent_context,
      pipe_arg,
      skip_parse,
-     raise_error) = pype.get_arguments(context)
+     raise_error,
+     loader) = pype.get_arguments(context)
 
     assert pipeline_name == 'pipe name'
     assert use_parent_context == 'parent context bool'
     assert skip_parse == 'skip parse'
     assert raise_error == 'raise err'
+    assert loader is None
```

Comment:
test_pype_get_arguments_all tests ALL possible inputs assigned, so in this case input should be
```
    context = Context({
        'pype': {
            'name': 'pipe name',
            'pipeArg': 'argument here',
            'useParentContext': 'parent context bool',
            'skipParse': 'skip parse',
            'raiseError': 'raise err',
            'loader': 'loader here'
        }
    })
```

and then `assert loader == 'loader here'`